packages/driver/src/multi-domain/domain_fn.ts

```diff
@@ -44,6 +79,12 @@ export const handleDomainFn = (Cypress: Cypress.Cypress, cy: $Cy, specBridgeComm
     cy.isStable(false, 'multi-domain-start')
   }
 
+  const setRunnableState = () => {
```

Comment:
This looks like a setter that would take an argument. Wonder if something like `setRunnableStateToPassed` helps represent this function more accurately.

Reply:
This makes sense to me, updated